knowledge-base/hierarchical-breadcrumb-dropdowns-blazor.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Hierarchical Breadcrumb in Blazor
+description: Learn how to customize the Breadcrumb for Blazor by embedding dropdowns within Breadcrumb items to optimize user interaction and functionality.
+type: how-to
+page_title: Hierarchical Breadcrumb in Blazor
+slug: hierarchical-breadcrumb-dropdowns-blazor
+tags: breadcrumb, blazor, customization, dropdown, itemtemplate, hierarchical
+res_type: kb
+ticketid: 1652751
+---
+
+## Environment
+
+| Product |
+| --- |
+| Breadcrumb for Blazor |
+
+## Description
+
+Incorporating custom components, such as dropdowns, within certain Breadcrumb "crumbs" can significantly enhance user interaction and functionality.
+
+This KB article also answers the following questions:
+- Is it possible to achieve a Breadcrumb hierarchical structure?
+- Is it possible to embed custom components within Breadcrumb crumbs?
+- How to use the `ItemTemplate` for adding dropdowns to Breadcrumb items?
+
+## Solution
+
+To embed dropdowns within the Breadcrumb "crumbs", utilize the `ItemTemplate`. This template allows for the customization of Breadcrumb items, enabling the inclusion of complex components such as dropdowns.
```

Comment:
* Link relevant docs resources in the instructions. It's easier for the reader to get more information from inline links, compared to "See Also" links that appear somewhere else. (It's fine to have the same links in the See Also section.)
* Use simpler sentence structure and wording.

```suggestion
To embed dropdowns in the Breadcrumb "crumbs", use an `ItemTemplate`. This template allows you to customize the Breadcrumb items, and include other components such as dropdowns.
```